package.json

```diff
@@ -35,7 +35,9 @@
     "Muiz Uvais (https://github.com/MuizU)",
     "Scott Wilson (https://github.com/scottjwilson)",
     "k-puchala (https:github.com/k-puchala)",
-    "Snehashish Ghosh (https:github.com/snehashish-ghosh98)"
+    "Snehashish Ghosh (https:github.com/snehashish-ghosh98)",
```

Comment:
The 'https:github...' is the error I meant. My apologies for not being crystal clear. 

It needs to be 'https://github...' for yours **and the two contributors above you** in the list. 

It's a very minor detail and was my sloppy mistake 😅

Reply:
@MizouziE No problem, I wasn't able to find this thing

Reply:
Ok, don't worry. I will merge the branch as it is and then make the corrections myself and tag you to show what I meant. Thanks all the same for the contribution!